src/reducer/videoReducer.jsx

```diff
@@ -0,0 +1,17 @@
+const initialVideoState = {
+  videos: [],
+  playlist: [],
+  categoryList: [],
+};
+
+const videoReducer = (state, action) => {
+  switch (action.type) {
+    case "INITIALIZE_VIDEOS":
+      return { ...state, videos: action.payload };
+
+    case "INITIALIZE_CATEGORIES":
+      return { ...state, categoryList: action.payload };
+  }
+};
```

Comment:
appreciation - the reducer implementation is good but is jsx format required for reducer ?